modules/detect_target/detect_target_factory.py

```diff
@@ -37,7 +39,6 @@ def create_detect_target(
                 device,
                 model_path,
                 override_full,
-                local_logger,
```

Comment:
Under `DetectTargetOption.ML_ULTRALYTICS` , add `local_logger` back to the arguments.